src/utils.sh

```diff
@@ -20,5 +22,25 @@ function kworkflow-help()
     "\tup,u - Wake up vm\n" \
     "\tcodestyle - Apply checkpatch on directory or file\n" \
     "\tmaintainers - Return the maintainers and the mailing list\n" \
+    "\tsearch - Search for expression on git log or directory\n" \
     "\thelp"
 }
+
+function search()
+{
+  if [[ "$#" -eq 0 ]]; then
+    complain "Expected path or 'log'"
+    exit 1
+  fi
+  if [[ "$1" == "log" ]]; then
+    git log | grep -nwI -e "$2"
```

Comment:
There is a better way to do it. Take a look at:

https://git-scm.com/docs/git-grep
https://stackoverflow.com/questions/2928584/how-to-grep-search-committed-code-in-the-git-history/2929502
https://stackoverflow.com/questions/1337320/how-to-grep-git-commit-diffs-or-contents-for-a-certain-word
https://stackoverflow.com/questions/7124914/how-to-search-a-git-repository-by-commit-message/7124949#7124949